Mathlib/Topology/ContinuousOn.lean

```diff
@@ -306,15 +295,6 @@ theorem nhdsWithin_compl_singleton_sup_pure (a : α) : 𝓝[≠] a ⊔ pure a =
   rw [← nhdsWithin_singleton, ← nhdsWithin_union, compl_union_self, nhdsWithin_univ]
 #align nhds_within_compl_singleton_sup_pure nhdsWithin_compl_singleton_sup_pure
 
-/- ./././Mathport/Syntax/Translate/Expr.lean:177:8: unsupported: ambiguous notation -/
-theorem nhdsWithin_prod_eq {α : Type _} [TopologicalSpace α] {β : Type _} [TopologicalSpace β]
```

Comment:
What happened to this decl?

Reply:
@jcommelin It was moved to `Topology.Constructions`.

Reply:
Apparently it moved to `Topology.Constructions`.